src/Compilers/CSharp/Test/Symbol/Compilation/SemanticModelAPITests.cs

```diff
@@ -3410,6 +3410,27 @@ static void F()
             var discarded2 = model.GetEnclosingSymbol(source.IndexOf(" => x"));
         }
 
+        [WorkItem(976, "https://github.com/dotnet/roslyn/issues/976")]
+        [Fact]
+        public void ConstantValueOfInterpolatedString()
+        {
+            var source = @"
+class Program
+{
+    static void Main(string[] args)
+    {
+        Console.WriteLine($""{DateTime.Now.ToString()}.{args[0]}"");
+    }
+}";
+
+            var comp = CreateCompilationWithMscorlib(source, options: TestOptions.ReleaseExe);
+            var tree = comp.SyntaxTrees.Single();
+            var model = comp.GetSemanticModel(tree);
+
+            var interp = tree.GetRoot().DescendantNodes().OfType<InterpolatedStringExpressionSyntax>().Single();
+            Assert.False(model.GetConstantValue(interp).HasValue);
```

Comment:
I believe that, if you were to use Assert.Null, it would tell you the non-null value if it ever failed.